### PR TITLE
Development

### PR DESCRIPTION
**Pull request by Aravindan07**

### CODE REVIEW

**Review by Jaynil1611 — CHANGES_REQUESTED** [review]
- Using consistent colors across componets
- Padding around image in card
- Image size & name in stacked list
- Use darker shade for gray text
- In mobile view, when you click on link close
- Fix CSS classes BEM navigation

**Review by sjsouvik — CHANGES_REQUESTED** [review]
Hey Aravindan!

Amazing work Aravindan, site is mobile responsive as well.

You can try to implement active section for side bar.
Also, add a footer, it'll look great

File: index.js [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+});
+
+openSidebarIcon.addEventListener("click", () => {
+	mobileSidebar.style.display = "flex";
+});
+
+modal.addEventListener("click", (e) => {
+	if (e.target === modal) {
+		modal.style.display = "none";
+	}
```
Comment (yogeshpendse) [inline index.js]: I'm giving a demo PR

**Review by yogeshpendse — CHANGES_REQUESTED** [review]
Demo PR

**Comment by asyriac** [comment]
1. There is an offset (the title of the section is not visible) on clicking any link in the sidebar. This is happening since the main section starts from the top of the viewport.

**Review by Jaynil1611 — APPROVED** [review]
You can merge the pull request. All changes have been accommodated.

**Comment by Aravindan07** [comment]
Merging PR from development to main

**Comment by khusharth** [comment]
- BEM: E stands for element.
```
 <img
	class="avatar__medium margin-lr-medium"
	src="image-url"
	alt="Avatar"
/>
```
- BEM example: .avatar__image .avatar--medium.

Here you should use modifier instead of the Element. : avatar--medium

- margin-lr-medium: -> mr4 ml4 
- The medium keyword might confuse the user. 

- 
```
<div class="alert alert__success alert__padding letter__spacing__medium">		
```
  -  Should remove alert__padding class.
  
- w__100 Not valid BEM: instead w-100 or w100 (Try not to use BEM in utilities)
  -  text__center -> text-center

- For stack list image: use fixed height width to make it rounded.

- Modal (can do later): Have close button at the bottom. Add some title and some body text.
-  (To do later): Add Js for Toast.

**Pull request merged**